packages/mrml-core/src/mj_accordion/render.rs

```diff
@@ -32,6 +32,22 @@ const STYLE: &str = r#"noinput.mj-accordion-checkbox { display: block! important
 "#;
 
 impl<'root> Renderer<'root, MjAccordion, ()> {
+    fn get_children(&self) -> Vec<&MjAccordionChild> {
+        fn folder<'root>(
+            mut acc: Vec<&'root MjAccordionChild>,
+            c: &'root MjAccordionChild,
+        ) -> Vec<&'root MjAccordionChild> {
+            match c {
+                MjAccordionChild::Fragment(f) => {
+                    acc.append(&mut f.children.iter().fold(Vec::new(), folder))
+                }
+                _ => acc.push(c),
+            }
+            acc
+        }
+        self.element.children.iter().fold(Vec::new(), folder)
+    }
```

Comment:
I think this should better be a `fn children_iter(&self) -> impl Iterator<Item = &MjAccordionChild>`, and in the parent component, we could use `flat_map`.

Reply:
I've used a flat_map with a boxed iterator, which should hopefully be an improvement.